vm/cmd/client/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/spacemeshos/go-spacemesh/common/types"
+	"github.com/spacemeshos/go-spacemesh/vm/cmd/client/cmd"
+)
+
+func main() {
+	types.SetNetworkHRP("atest")
```

Comment:
That could be taken from CLI parameter defaulting to this value, but from the other hand - maybe it's safer to keep it in this hardcoded shape for right now...

Reply:
I prefer it hardcoded so nobody uses it against the mainnet or our "normal" testnet.